samples/vue/src/AppRoot.vue

```diff
@@ -29,7 +29,7 @@ export default {
         // to show a loading/switching screen when language is being changed.
         this.languageIsChanging = true;
 
-        dictionaryService.fetchDictionaryData(language).then((phrases) => {
+        dictionaryServiceFactory.fetchDictionaryData(language).then((phrases) => {
```

Comment:
Don't we need to create a dictionary service instance first?
```suggestion
        const dictionaryServiceInstance = dictionaryServiceFactory.create();
        dictionaryServiceInstance.fetchDictionaryData(language).then((phrases) => {
```